internal/core/data/v2/controller/http/event.go

```diff
@@ -416,3 +416,29 @@ func (ec *EventController) EventsByTimeRange(w http.ResponseWriter, r *http.Requ
 	utils.WriteHttpHeader(w, ctx, statusCode)
 	pkg.Encode(response, w, lc)
 }
+
+func (ec *EventController) ReadingTotalCount(w http.ResponseWriter, r *http.Request) {
```

Comment:
Move to new `reading.go` since focus is on Reading, not Event. Other Reading APIs will also go into `reading.go`

Reply:
Moved.